jfrog_platform_cloud/.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @danielmkn
```

Comment:
You will need to add your integration name and github handle in alphabetical order within this file. Please edit the file to include the above. To edit via web ui: 

1. Go to your forked repository > .github > CODEOWNERS file
2. Edit the file by clicking the pencil icon in the top right 
3. Add your integration name and github handle in alphabetical order
4. Click Commit changes directly to the master branch

Reply:
Removed this file and added requested changes to `.github/CODEOWNERS`